core-web/apps/dotcms-block-editor/src/test.ts

```diff
@@ -7,6 +7,7 @@ import {
     platformBrowserDynamicTesting
 } from '@angular/platform-browser-dynamic/testing';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Who's crying about this? lint, typescript or vscoe?

Reply:
Removed. The editor, lint, build and test works fine.